addons/website_slides/models/slide_channel.py

```diff
@@ -33,6 +34,7 @@ class ChannelUsersRelation(models.Model):
     channel_visibility = fields.Selection(related='channel_id.visibility')
     channel_enroll = fields.Selection(related='channel_id.enroll')
     channel_website_id = fields.Many2one('website', string='Website', related='channel_id.website_id')
+    next_slide_id = fields.Many2one('slide.slide', string="Next Lesson")
```

Comment:
why ? (there's no mention in the commit of this new feature)

IMO, it should be a computed non-stored field

Reply:
If we want to do so, we have to recompute everything for this.
is that okay for you?

Thanks

Reply:
yes ofc